frontend/packages/noobaa-storage-plugin/integration-tests/tests/overviewDashboard.scenario.ts

```diff
@@ -0,0 +1,107 @@
+import { execSync } from 'child_process';
+import { browser, ExpectedConditions as until } from 'protractor';
+import { appHost, testName } from '@console/internal-integration-tests/protractor.conf';
+import { untilNoLoadersPresent } from '@console/internal-integration-tests/views/crud.view';
+import {
+  obcCount,
+  efficiencyValue,
+  savingsValue,
+  healthOfMCG,
+  resourceProviders,
+  noobaaCount,
+} from '../views/noobaaDashboardPage.view';
+
+const testBucket = {
+  apiVersion: 'objectbucket.io/v1alpha1',
+  kind: 'ObjectBucketClaim',
+  metadata: {
+    namespace: testName,
+    name: 'test-bucket',
+  },
+  spec: {
+    ssl: false,
+    storageClassName: 'openshift-storage.noobaa.io',
+    generateBucketName: 'test-bucket',
+  },
+};
+
+const loadAndWait = async () => {
+  await browser.get(`${appHost}/dashboards/object-service`);
+  await browser.wait(until.and(untilNoLoadersPresent));
+};
+
+describe('Tests Buckets Card', () => {
+  beforeEach(async () => {
+    await loadAndWait();
+  });
+
+  it('Tests at least one noobaa bucket is present', async () => {
+    await browser.wait(until.visibilityOf(noobaaCount));
+    const text = await noobaaCount.getText();
+    const [count, ,] = text.split(' ');
+    expect(Number(count)).toBeGreaterThanOrEqual(1);
+  });
+
+  it('Creates an Object Bucket Claim and tests equality', async () => {
+    await browser.wait(until.visibilityOf(obcCount));
+    let text = await obcCount.getText();
+    const initialCount = text.substring(0, 1);
+    await execSync(`echo '${JSON.stringify(testBucket)}' | kubectl create -n ${testName} -f -`);
+    await browser.sleep(10000);
```

Comment:
It would be must nicer to wait for some condition. You can take a look here how simple waiting method can look: https://github.com/openshift/console/blob/41bb8cb8354ab91d18377535c17e070c7e5a0c75/frontend/packages/console-shared/src/test-utils/utils.ts#L135
and here how it's used:
https://github.com/openshift/console/blob/41bb8cb8354ab91d18377535c17e070c7e5a0c75/frontend/packages/kubevirt-plugin/integration-tests/tests/vm.actions.scenario.ts#L130